metabase/metabase
# Repro #25988: Cannot reference nested question as native subquery

### Status
PENDING REVIEW

### What does this PR accomplish?
- Reproduces #25988 

### How to test this manually?
- `yarn test-cypress-open`
- `frontend/test/metabase/scenarios/native/native_subquery.cy.spec.js`
- Replace `it.skip()` with `it.only()` to run the test in isolation
- The test should fail until the related issue is fixed

### Additional notes:
- Once the issue is fixed, please remove the `.skip` part (unskip the test completely)
- Make sure the test is passing and
- Merge it together with the fix

### Screenshots:
![image](https://user-images.githubusercontent.com/31325167/217374875-9e50cae7-900e-4488-87f1-45bee47ca706.png)
